SConstruct

```diff
@@ -27,70 +46,124 @@ def getExecutableCmd(package, script, *args, directory=None):
-# defects = runPipeline("defects", ct)
-# dark = runPipeline("dark", bias)  # defects?
-# gain = runPipeline("gain", dark)
-# bfkernel = runPipeline("bfkernel", gain)
-# flat = runPipeline("flat", bfkernel)
-# fringe = runPipeline("fringe", flat)
-
-# science = runPipeline("science", flat)
-
-
+                                     "-d", os.path.join(TESTDATA_ROOT, "raw", "2020-01-28"),
+                                     "--output-run", RAW_COLLECTION),
+                    Touch(".scons_ingest")
+                ])
+ingest = env.Alias("ingest", raws)
+
+# Bias
+bias = env.Command(os.path.join(REPO_ROOT, 'ci_cpp_bias'), ".scons_ingest",
+                   [getPipeTaskCmd('bias', exposureDict['biasExposures'],
+                                   biasCollection, 'cpBias.yaml'),
+                    getCertifyCmd('bias')])
+env.Alias('bias', bias)
+
+# Dark
+dark = env.Command(os.path.join(REPO_ROOT, 'ci_cpp_dark'), bias,
+                   [getPipeTaskCmd('dark', exposureDict['darkExposures'],
+                                   genCollection, 'cpDark.yaml'),
+                    getCertifyCmd('dark')])
+env.Alias('dark', dark)
+
+# Flat
+flat = env.Command(os.path.join(REPO_ROOT, 'ci_cpp_flat'), dark,
+                   [getPipeTaskCmd('flat', exposureDict['flatExposures'],
+                                   genCollection, 'cpFlat.yaml'),
+                    getCertifyCmd('flat')])
+env.Alias('flat', flat)
+
+# Crosstalk
+crosstalk = env.Command(os.path.join(REPO_ROOT, f'ci_cpp_crosstalk'), dark,
+                        [getPipeTaskCmd('crosstalk', exposureDict['scienceExposures'],
+                                        genCollection, 'measureCrosstalk.yaml'),
+                         getCertifyCmd('crosstalk')])
+env.Alias('crosstalk', crosstalk)
+
+# Science
+scienceYaml = os.path.join(PKG_ROOT, 'pipelines', 'runIsr.yaml')
+sciExposures = ",".join([str(vv) for vv in exposureDict['scienceExposures']])
+science = env.Command(os.path.join(REPO_ROOT, 'science'), [crosstalk, flat],
+                      [getExecutableCmd('ctrl_mpexec', 'pipetask',
+                                        'run -j', str(num_process),
+                                        f'-d "detector=0 AND exposure IN ({sciExposures})"',
+                                        f"-b {REPO_ROOT}/butler.yaml", f"-i {genCollection}",
+                                        f"-o science",
+                                        f"-p {scienceYaml}",
+                                        '--register-dataset-types'
+                                    )])
+env.Alias('science', science)
+
+# Tests and bookkeeping
 everything = [butler, instrument, raws] 
-# everything.extend([bias, dark, flat, science])
-# everything.extend([linearity, ct, defects, gains])
+everything.extend([bias, dark, flat, crosstalk, science])
+# These need gen3 conversion first.
+# everything.extend([ptc, bfk, linearity, defects, gains])
+
+tests = [env.Command(f"test_{name}", ['DATA/science'],
```

Comment:
Why can't you put the standard Scons file in the tests directory and use the standard test target that already knows how to run tests using pytest?